liby2/src/genericfrontend.cc

```diff
@@ -657,42 +651,19 @@ main (int argc, char **argv)
 
 // FIXME the whole option parsing sucks **** !
 
-    // Check, if debugger should be enabled
+    // old backward compatibility stuff
```

Comment:
I think we should simply drop this. If anyone were still using `--debugger` they would expect the YCP thing. We are removing it so let's remove this too.